### ISSUE TITLE
Security issues for WordPress ULS Upstream

### ISSUE THREAD

**Issue by jgreidy** [cul-it/uls#1202]
These are from GitHub Dependabot

https://github.com/cul-it/ci-jenkins-pantheon-wordpress/security/dependabot

![Screenshot 2023-08-15 at 1 44 48 PM](https://github.com/cul-it/uls/assets/5996967/7b1c3132-58b5-4a8f-9e01-26920a41c551)

**Comment by sarah-cul** [comment]
I believe that issues was addressed. But currently: 

![image](https://github.com/cul-it/uls/assets/144364984/aaa34d2b-b284-4253-a2f4-576c3abd4ed1)

**Comment by sarah-cul** [comment]
@djc88 @ebtoner This particular issue was addressed some time ago, but there are other [dependabot](https://github.com/cul-it/ci-jenkins-pantheon-wordpress/security/dependabot) warnings that have gone unaddressed. This applies to the upstream unit sites only, I've ever looked at these before, Jim must have taken care of this. We could take a look at these in the next upstream? or...

@JesiBuell79 should we put this work on hold until we know the plan?

**Comment by sarah-cul** [comment]
related: https://github.com/cul-it/uls/issues/1201

**Comment by JesiBuell79** [comment]
Yes, @sarah-cul - I think this should wait until we know what we're going to do unless there's an immediate concern!
